TinCan/ActivityDefinition.cs

```diff
@@ -26,13 +35,13 @@ public class ActivityDefinition : JsonModel
         public LanguageMap name { get; set; }
         public LanguageMap description { get; set; }
         public Extensions extensions { get; set; }
-        //public InteractionType interactionType { get; set; }
-        //public List<String> correctResponsesPattern { get; set; }
-        //public List<InteractionComponent> choices { get; set; }
-        //public List<InteractionComponent> scale { get; set; }
-        //public List<InteractionComponent> source { get; set; }
-        //public List<InteractionComponent> target { get; set; }
-        //public List<InteractionComponent> steps { get; set; }
+        public String interactionType { get; set; }
```

Comment:
I'd prefer to make `InteractionType` its own class, possibly based on an enum or something like it. That would follow the convention from `TinCanJava` and make matching the types (since there is a set list) easier. See https://github.com/RusticiSoftware/TinCanJava/blob/master/src/main/java/com/rusticisoftware/tincan/InteractionType.java